### PR TITLE
PyTorch sentiment analysis example

### PR DESCRIPTION
**Pull request by chicm-ms**
A sentiment analysis example which demonstrates:
1. Customizing NLP dataset processing with torchtext
2. Implementation of RNN model with simple attention mechanism
3. PyTorch train process, backpropagation and calculating accuracy

### CODE REVIEW

**Comment by shishaochen** [comment]
Please add a README.md first.
By the way, could you provide a script for inference as well?
In the README, a customer may know how to prepare his dataset, how to do inference with the trained model.

**Comment by chicm-ms** [comment]
Inference code added.
Readme will be added later for all pytorch examples.
